backend/api/database.py

```diff
@@ -85,9 +94,21 @@ def update_district(db: Session, district):
 
 def get_moodle_settings(db: Session):
     moodle_settings = db.query(MoodleSetting).all()
+    if len(moodle_settings) == 0:
+        raise NoResultFound
```

Comment:
Raising here versus returning `[]` has two potential nits:

1. In the calling code in `main.py`, it seems to assume this will return `[]` vs raise an exception on `/admin/settings/moodle` (e.g. there is no `try / except`). 
2. In general, looking at `database.py` we seem to return `None` / `[]` on read queries and only raise on updates where the corresponding row doesn't exist.

Maybe this change was accidental, though, since it doesn't actually relate to changes in this PR? :man_shrugging:

Reply:
:thinking: - I have the code at `98a4f5f0e5210bdb301eee1744e228b8eb08c771` and still see this. Maybe the change didn't get pushed up?

Reply:
Odd - I could have sworn I pushed the change yesterday. I just pushed this update.